server/explorer_server.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"google.golang.org/grpc"
+	"log"
+	"net"
+
+	pb "github.com/iotexproject/iotex-election/explorer_pb"
+)
+
+func main() {
+	lis, err := net.Listen("tcp", ":8089")
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
+	}
+	// Creates a new gRPC server
+	s := grpc.NewServer()
+	pb.RegisterExplorerServer(s, &pb.server{})
```

Comment:
1. pb.server is supposed to be a private struct in pb.
2. To construct pb.server, we need to construct "committee.Committee" and feed it to server.

Reply:
Need to look more into code. For now it was just a simple work on conversion of idl file to protobuf file and add a simple method. I hope I can do it after a proper code go through. Will try to do it asap.